core/utils/poly-look/src/react/chips/chips.jsx

```diff
@@ -1,21 +1,131 @@
-import React from "react";
+import React, { useState } from "react";
 
 import "./chips.css";
 
-const Chips = ({ chipsContent, activeChips, onChipClick, theme }) => {
+const chipsId = (e) => e.id || e.title || e;
+
+const othersId = "others";
+const allId = "all";
+
+/**
+ *
+ * Chips group
+ * @param {string[] || Object[]} chipsContent -  Content of the Chips
+ * @param {Object} [chipsContent.id] - Id of the chip (if chipsContent is an array of strings they act as id)
+ * @param {Object} [chipsContent.title] - Alternative for id of the chip (if chipsContent is an array of strings they act as id)
+ * @param {Object} [chipsContent.translation] - Translation of the chip (if chipsContent is an array of strings they act as translation)
+ * @param {string[]} defaultActiveChips - Chips that are active on load
+ * @param {callback} onChipClick - Chips onClick function (id of clicked chip, all active chips ids)
+ * @param {boolean = true} [exclusive] - Determines whether chips are active exclusively
+ * @param {string} [theme] - Sets the theme in this component (preferably done in parent component)
+ * @param {Object} [all] - Indicated whether an all chip exists (exclusive to the other chips, activates all)
+ * @param {string = "All"} [all.translation] - Translation for chips text ("All")
```

Comment:
Please check (and possibly merge) #585 before merging this. JSDoc does not seem to like this syntax, preferring 

    @param {string} [all.translation  = "All" ]

instead

Reply:
Done!